### PR TITLE
Fix layout issues in 'decohelp - Sales Training' and 'Docs'

### PR DESCRIPTION
**Pull request by devartes**

### CODE REVIEW

File: components/decohelp/pages/ui/ForwardBackButtons/ForwardBackButtons.tsx [hunk 1]
```diff
@@ -1,58 +1,80 @@
...
+  useEffect(() => {
+    const encodedURL = window.location.href;
+    const decodedURL = decodeURIComponent(encodedURL);
+
+    const links = document.querySelectorAll("aside a");
+    const linkArray = Array.from(links);
+
+    const currentIndex = linkArray.findIndex((link) =>
+      decodedURL.includes(link.getAttribute("href") || "")
+    );
```
Comment (matheusgr) [inline components/decohelp/pages/ui/ForwardBackButtons/ForwardBackButtons.tsx:33]: That includes test may match with underisable links.

```suggestion
    const currentIndex = linkArray.map((x, index) => ({value: x, index})).toSorted((x,y) => x.value.length - y.value.length).find((link) =>
      decodedURL.includes(link.getAttribute("href") || "")
    )?.index;
```

File: components/decohelp/pages/ui/ForwardBackButtons/ForwardBackButtons.tsx [hunk 2]
```diff
@@ -1,58 +1,80 @@
...
+  const [next, setNext] = useState<PageButton>({
+    title: "",
+    href: "",
+  });
+
+  const isEnglish = window.navigator.language.startsWith("en");
+  const [isEnglishState, setIsEnglishState] = useState(isEnglish);
+
+  useEffect(() => {
+    const encodedURL = window.location.href;
```
Comment (matheusgr) [inline components/decohelp/pages/ui/ForwardBackButtons/ForwardBackButtons.tsx:25]: may cause some issues with query parameters and hashs.

```suggestion
    const encodedURL = window.location.href.split(window.location.search||window.location.hash||/[?#]/)[0];
```

Comment (devartes) [reply components/decohelp/pages/ui/ForwardBackButtons/ForwardBackButtons.tsx]: image
done

File: components/decohelp/pages/ui/ForwardBackButtons/ForwardBackButtons.tsx (hunk 1 above)
Comment (devartes) [reply components/decohelp/pages/ui/ForwardBackButtons/ForwardBackButtons.tsx]: image

I made a modification in the code to resolve the type errors.
_Property 'length' does not exist on type 'Element' and Property 'getAttribute' does not exist on type '{ value: Element; index: number; }'._

**Review by matheusgr — APPROVED** [review]

**Pull request merged**